mirrord/layer/src/exec.rs

```diff
@@ -19,6 +25,8 @@ use crate::{
     replace,
 };
 
+static MAX_ARGC: usize = 16;
```

Comment:
why 16?

Reply:
Just seemed high enough to me. When there are more args we bypass the argv interception.
Do you think it should be greater? How much?

Reply:
254 and I think it needs to be a const not static

Reply:
cool, thanks!